Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ipc-channel"
-version = "0.14.0"
+version = "0.14.1"
```

Comment:
The `RouterMsg` enum is added one more variant. I think it might be good to upgrade the minor version instead 👀?

Reply:
The enum is private, so I don't think it changes the api?

Reply:
Ahh, I see; didn't notice that. sorry, then ignore my comment >_<